File: Projeto_2/PROJETO 2 - IVAN/src/cliente.py
class Cliente:
    def __init__(self, nome, cpf):
        self.nome = nome
        self.__cpf = cpf

    def get_cpf(self):
        return self.__cpf

    def exibir_informacoes(self):
        print(f"Cliente: {self.nome}, CPF: {self.__cpf}")

File: Projeto_2/projeto_2 - Dan Mendes de Souza Nogueira Ribeiro/src/Models/cliente.cpp
#include <iostream>
#include <string>
#include "cliente.h"


    void cliente::setNome(std::string nomeAdicionar) { nome = nomeAdicionar; };

    void cliente::setCPF(std::string cpfAdicionar) { cpf = cpfAdicionar; };

    std::string cliente::getCPF() { return cpf; };

    std::string cliente::getNome() { return nome; };
